CPS-?/README.md

```diff
@@ -0,0 +1,75 @@
+---
+CPS: ?
+Title: Domain Information Resolving Standard
+Status: Proposed
+Category: Wallets
```

Comment:
```suggestion
Category: Tools
```
Based on your observation & a general view I'd re-cast this as `Tools` because it addresses dApp and UX issues as well as `Wallets`.  From [CIP-0001 > Categories](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#categories):
> Tools | A broad category for ecosystem tools not falling into any other category.